plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +53,52 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            Double range;
+
+            try {
+                range = Math.pow(Double.parseDouble(args[1]), 2);
+            }
+            catch (Exception exception) {
+                range = this.config.inviteCommandAllMaxRange;
```

Comment:
Wysyłaj komunikat, że nie poprawna liczba i returnuj

Reply:
Interesująca opcja, faktycznie przegapiłem — niejawnym fallback jest okropny dla UX, ale sam catch do zaorania.

Reply:
Nie lepiej zrobić wiadomość typu: "Zapraszam wszystkich graczy w pobliżu x do gildii!"? Wtedy UX będzie lepsze, ponieważ radius będzie znany

Reply:
Nie rozumiem, jaki to ma związek z komunikatem o błędzie, dotyczącym parametrów komendy?